tests/ml_test.py

```diff
@@ -0,0 +1,33 @@
+#sanity tests for the ml functions,
+
+
+import unittest
+import os
+import numpy as np
+from resurfemg.data_connector.tmsisdk_lite import Poly5Reader
+from resurfemg.machine_learning.ml import save_ml_output
+from resurfemg.preprocessing.filtering  import emg_bandpass_butter
+from tempfile import TemporaryDirectory
+
+
+sample_emg = os.path.join(
+    os.path.abspath(os.path.dirname(os.path.dirname(__file__))),
+    'not_pushed',
+    'Test_lung_data',
+    '2022-05-13_11-51-04',
+    '002',
+    'EMG_recording.Poly5',
+)
+
+class TestMl(unittest.TestCase):
+
+
+    def test_save_ml_output(self):
+        sample_read= Poly5Reader(sample_emg)
+        sample_emg_filtered = emg_bandpass_butter(sample_read, 1, 500)
+        with TemporaryDirectory() as td:
+            our_path = os.path.join(td,'outy.npy')
+            emg_saved = save_ml_output(sample_emg_filtered, our_path , force=True)
```

Comment:
This line throws a linting error:
Assigning result of a function call, where the function returns None

i.e., save_ml_output doesn't return anything and emg_saved is also not used in the rest of the test. Consider removing the "emg_saved = ":
save_ml_output(sample_emg_filtered, our_path , force=True)